services/galaxytoolshed/galaxytoolshed-version.service.js

```diff
@@ -0,0 +1,85 @@
+import { NotFound } from '../index.js'
+import { renderVersionBadge } from '../version.js'
+import GalaxyToolshedService from './galaxytoolshed-base.js'
+
+export class GalaxyToolshedVersion extends GalaxyToolshedService {
+  static category = 'version'
+  static route = {
+    base: 'galaxytoolshed/v',
+    pattern: ':repository/:owner/:tool?/:requirement?',
+  }
+
+  static examples = [
```

Comment:
We in the process of migrating our examples to the OpenAPI format (see #9285). Could. I please get you to update your PR accordingly? https://github.com/badges/shields/blob/master/services/cirrus/cirrus.service.js is a simple example using the new format.

Reply:
`examples` were migrated to `openApi`